.circleci/config.yml

```diff
@@ -111,7 +112,7 @@ jobs:
 
       - save_cache:
           name: "Cache Nix Store Paths"
-          key: paymentserver-nix-store-v3-{{ checksum "nixpkgs.rev" }}-{{ checksum "nix/challenge-bypass-ristretto-repo.nix" }}
+          key: paymentserver-nix-store-v5-{{ checksum "nixpkgs.rev" }}-{{ checksum "nix/challenge-bypass-ristretto-repo.nix" }}
```

Comment:
Having saxtons configured as a binary substituter and using the same revision of nixpkgs, I wonder if this cache adds anything at all now - saxtons should give us the challenge-bypass-ristretto binaries, I think?

Reply:
And getting the packages from a nix binary substituter should be a lot faster than using a CircleCI cache of all of /nix/store.

Reply:
I'd be inclined to replace it with cachix (at least assuming we don't soon move this to whetsone). We do still need to build all of the haskell packages, when we update something like the stackage version, but not GHC.

Reply:
(But I thought this was a large enough change on its own, so didn't do anything but bump the cache revision here)